charts/apps/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: graph
 description: ArgoCD Apps used to deploy the Graph
 type: application
-version: 0.1.0
+version: 0.1.1
```

Comment:
This feels like a breaking change given the structure of the `values.yaml` changes :sweat_smile: